tests/Core/Components/Divider/FluentDividerTests.razor

```diff
@@ -0,0 +1,76 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using Xunit;
+@inherits TestContext
+
+@code
+{
+    public FluentDividerTests()
+    {
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Services.AddFluentUIComponents();
+    }
+
+    [Fact]
+    public void FluentDivider_Default()
+    {
+        // Arrange && Act
+        var cut = Render(@<FluentDivider>Hello World</FluentDivider>);
+
+        // Assert
+        cut.Verify();
+    }
+
+    [Theory]
+    [InlineData(DividerAlignContent.Center, "center")]
+    [InlineData(DividerAlignContent.Start, "start")]
+    [InlineData(DividerAlignContent.End, "end")]
+    public void FluentDivider_AlignContent(DividerAlignContent value, string expectedValue)
+    {
+        // Arrange && Act
+        var cut = Render(@<FluentDivider AlignContent="@value">My content</FluentDivider>);
+
+        // Assert
+        var attribute = cut.Find("fluent-divider").GetAttribute("align-content") ?? string.Empty;
+        Assert.Equal(expectedValue, attribute);
+    }
+
+    [Theory]
+    [InlineData(DividerAppearance.Default, "")]
+    [InlineData(DividerAppearance.Subtle, "subtle")]
+    [InlineData(DividerAppearance.Brand, "brand")]
+    [InlineData(DividerAppearance.Strong, "strong")]
```

Comment:
Same, you need to test `null` and `(DividerAppearance)999` which are possible values